sql/branch.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN;
+
+    CREATE TABLE package_managers
+    ( id        bigserial       PRIMARY KEY
+    , name      text            NOT NULL UNIQUE
+     );
+
+    CREATE TABLE packages
+    ( id                    bigserial                   PRIMARY KEY
+    , package_manager_id    bigint                      NOT NULL REFERENCES package_managers ON UPDATE CASCADE ON DELETE RESTRICT
```

Comment:
Pretty sure we don't need the `ON UPDATE ... ON DELETE` since we're referencing an immutable field.